stdlib/src/builtin/string.mojo

```diff
@@ -570,11 +570,44 @@ fn _is_ascii_lowercase(c: UInt8) -> Bool:
 # ===----------------------------------------------------------------------===#
 
 
+fn isspace_python(s: String) -> Bool:
```

Comment:
Can you please move this to a hidden method on the struct, i.e. `String._isspace_python()`?